src/atoms/Input/Input.tsx

```diff
@@ -13,11 +13,11 @@ export const Input = styled(Typography)`
   font-size: ${scale(0)};
   font-weight: bold;
   ${padding(scale(-1), scale(0))};
-  transition: border ${transitionDuration};
+  transition: border ${transitionDuration}, box-shadow ${transitionDuration};
 
   :focus {
-    border-color: ${props => props.theme.primary};
     outline: none;
+    box-shadow: 0 0 0 2px white, 0 0 0 4px ${props => props.theme.primary};
```

Comment:
Same about pixels here, it would be nice to also store this in `Theme.tsx` so you only have to change it once in the future

Reply:
This is a good idea!